ui/src/app/settings/components/SettingsTable/SettingsTable.js

```diff
@@ -35,11 +35,20 @@ export const SettingsTable = ({
         ) : (
           <div className="p-table-actions__space-left"></div>
         )}
-        {buttons.map(({ label, url }) => (
-          <Button element={Link} to={url} key={url}>
-            {label}
-          </Button>
-        ))}
+        {buttons.map(({ label, url, disabled = false, tooltip }) =>
```

Comment:
I wonder if it would be a good idea to include a tooltip prop for the Button component itself? I've noticed it's quite a common pattern in MAAS and RBAC to disable a button (or checkbox) and have a tooltip display why.